contributing/configlet/generating-documents.md

```diff
@@ -1,6 +1,6 @@
 # Configlet generating documents
 
-The secondary use of [configlet](./README.md) is generating documents.
+The secondary use of [configlet](../) is generating documents.
```

Comment:
How is this link rendered as it doesn't point to a document anymore.

Reply:
Documents != URLs.

So `contibuting/configlet` will render a document (per the config.json) so this document having a link to `../` points to that node.